app/views/users/show.html.erb

```diff
@@ -161,7 +161,7 @@
         <% if user_signed_in? %>
           <%= javascript_packs_with_chunks_tag "profileDropdown", defer: true %>
         <% end %>
-        <a href="/report-abuse?url=<%= user_url(@user) %>">Report Abuse</a>
+        <span class="report-abuse-link-wrapper" data-path="/report-abuse?url=<%= user_url(@user) %>"></span>
```

Comment:
Ah, are we switching from `<a>` to `<span data-path=...>` because the custom data attribute won't be indexed by Google?

Reply:
Yeah. Basically we're taking a span and including the necessary data (the path) and then only creating the link for user-triggered action.

Google sees this as a useless span instead of a div. The dumb spider isn't smart enough to just ignore these kinds of links 🤪